doc/source/feature_overview.md

```diff
@@ -0,0 +1,263 @@
+in {attr}`~geoutils.Raster.bounds`. Using {func}`~geoutils.Raster.polygonize` allows to generate a {class}`~geoutils.Vector` from a {class}`~geoutils.Raster`,
+and the other way around for {func}`~geoutils.Vector.rasterize`.
+
+```{list-table}
+   :widths: 30 30 30
+   :header-rows: 1
+
+   * - **Class**
+     - {class}`~geoutils.Raster`
+     - {class}`~geoutils.Vector`
+   * - Warping/Reprojection
+     - {func}`~geoutils.Raster.reproject`
+     - {func}`~geoutils.Vector.reproject`
+   * - Cropping/Clipping
+     - {func}`~geoutils.Raster.crop`
+     - {func}`~geoutils.Vector.crop`
+   * - Rasterize/Polygonize
+     - {func}`~geoutils.Raster.polygonize`
+     - {func}`~geoutils.Vector.rasterize`
+```
+
+All methods can also be passed any number of georeferencing arguments such as {attr}`~geoutils.Raster.shape` or {attr}`~geoutils.Raster.res`, and will
+naturally deduce others from the input {class}`~geoutils.Raster` or {class}`~geoutils.Vector`, much as in [GDAL](https://gdal.org/)'s command line.
+
+
+## Higher-level analysis tools
+
+GeoUtils also implements higher-level geospatial analysis tools for both {class}`Rasters<geoutils.Raster>` and {class}`Vectors<geoutils.Vector>`. For
+example, one can compute the distance to a {class}`~geoutils.Vector` geometry, or to target pixels of a {class}`~geoutils.Raster`, using
+{func}`~geoutils.Vector.proximity`.
+
+As with the geospatial handling functions previously listed, many analysis functions can take a {class}`~geoutils.Raster` or {class}`~geoutils.Vector` as a
+**reference to utilize** during the operation. In the case of {func}`~geoutils.Vector.proximity`, passing a {class}`~geoutils.Raster` serves as a reference
+for the georeferenced grid on which to compute the distances.
+
+```{code-cell} ipython3
+# Compute proximity to vector on raster's grid
+rast_proximity_to_vec = vect.proximity(rast)
+```
+
+```{note}
+Right now, the array {attr}`~geoutils.Raster.data` of `rast` is still not loaded. Applying {func}`~geoutils.Raster.crop` does not yet require loading,
+and `rast`'s metadata is sufficient to provide a georeferenced grid for {func}`~geoutils.Vector.proximity`. The array will only be loaded when necessary.
+```
+
+## Quick plotting
+
+To facilitate the analysis process, GeoUtils includes quick plotting tools that support multiple colorbars and implicitly add layers to the current axis.
+Those are build on top of {func}`rasterio.plot.show` and {func}`geopandas.GeoDataFrame.plot`, and relay any argument passed.
+
+```{seealso}
+GeoUtils' plotting tools only aim to smooth out the most common hassles when quickly plotting raster and vectors.
+
+For advanced plotting tools to create "publication-quality" figures, see [Cartopy](https://scitools.org.uk/cartopy/docs/latest/) or
+[GeoPlot](https://residentmario.github.io/geoplot/index.html).
+```
+
+The plotting functionality is named {func}`~geoutils.Raster.show` everywhere, for consistency. Here again, a {class}`~geoutils.Raster` or
```

Comment:
This makes me think. Initially we decided to stick to rasterio's naming convention, because we are more raster oriented. But "plot" would be more intuitive, no?

Reply:
Yes, maybe we could change to `plot()` everywhere... Rasterio is the only one using `show()`

Reply:
Thoughts @atedstone?

Reply:
I agree. I find `show()` weird, let's change to `plot()`.

Reply:
OK, opening an issue, I'll do this in a separate PR!